manifest.md

```diff
@@ -60,17 +60,26 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
-- **`annotations`** *string-string hashmap*
+- **`annotations`** *string-string map*
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
-    Common annotation keys include:
-    * **created** date on which the image was built (string, timestamps type)
-    * **authors** contact details of the people or organization responsible for the image (freeform string)
-    * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
-    * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
-
+    Annotations MUST BE a key-value map where both the key and value MUST be strings.
+    While the value MUST be present, it MAY be an empty string.
+    Keys MUST be unique within this map, and best practice is to namespace the keys.
+    Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
```

Comment:
On Wed, Sep 07, 2016 at 08:36:09AM -0700, Jonathan Boulle wrote:
> >  
> >      This OPTIONAL property contains arbitrary metadata for the manifest list.
> > -    Annotations is a key-value, unordered hashmap.
> > -    Keys are unique, and best practice is to namespace the keys.
> > -    Common annotation keys include:
> > -    * **created** date on which the image was built (string, timestamps type)
> > -    * **authors** contact details of the people or organization responsible for the image (freeform string)
> > -    * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
> > -    * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
> > -
> > +    Annotations MUST BE a key-value map where both the key and value MUST be strings.
> > +    While the value MUST be present, it MAY be an empty string.
> > +    Keys MUST be unique within this map, and best practice is to namespace the keys.
> > +    Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
> > +    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
>
> what do you mean by subsequent specifications? how about just
> "other"? (wouldn't we expect that subsequent versions of this
> specification would honour the same ones?)

+1 to avoiding “subsequent specifications”.  I recommended using “…
are reserved for future versions of the specification.” [1].

[1]: https://github.com/opencontainers/runtime-spec/pull/510#discussion_r77291491